pkg/config/setup/process.go

```diff
@@ -205,6 +205,9 @@ func setupProcesses(config pkgconfigmodel.Setup) {
 
 	procBindEnvAndSetDefault(config, "process_config.language_detection.grpc_port", DefaultProcessEntityStreamPort)
 
+	// Connection Capacity Check configuration
+	procBindEnvAndSetDefault(config, "process_config.connections_capacity_check_interval", 30*time.Second)
```

Comment:
Could we move this to a child of `process_config`, like `process_config.connections` or similar?

Reply:
done, good suggestion